features/support/testing_dsl.rb

```diff
@@ -797,8 +798,11 @@ module Shell
 ERRORFORMAT
 
       def self.run(command, allow_failures = false)
-        output = `#{command} 2>&1`
-        status = $CHILD_STATUS
+        env = ENV.to_h.dup
+        env.delete('BUNDLE_GEMFILE')
```

Comment:
This might be unneeded, but the BUNDLE_GEMFILE was leaking in. I'll test tomorrow if it can be removed.